tools/build-toolchain.sh

```diff
@@ -124,6 +184,29 @@ CFLAGS_FOR_TARGET="-G0 -O2" CXXFLAGS_FOR_TARGET="-G0 -O2" ../"gcc-$GCC_V"/config
   --disable-threads \
   --disable-win32-registry \
   --disable-nls \
-  --with-system-zlib
+  --with-system-zlib \
+  $CROSS_COMPILE_FLAGS
+make -j "$JOBS"
+make install || sudo make install || su -c "make install"
+echo "Finished Compiling gcc-$GCC_V for MIPS N64 (pass 2) outside of the source tree"
+
+if [ "$MAKE_V" != "" ]; then
+echo "Compiling make-$MAKE_V"
+# As make is otherwise not available on Windows
+cd ../"make-$MAKE_V"
+  ./configure \
+    --prefix="$INSTALL_PATH" \
+    --disable-largefile \
+    --disable-nls \
+    --disable-rpath \
+    $CROSS_COMPILE_FLAGS
 make -j "$JOBS"
 make install || sudo make install || su -c "make install"
+echo "Finished Compiling make-$MAKE_V"
+fi
```

Comment:
I don't think we should build make here, also because it's not enough: n64.mk also requires bash and other tools. Maybe we can just require a shell like msys to be installed on Windows? msys already contains make and other required tools.

Reply:
My argument here would be that if you have to use MSYS (or an equivalent environment), it would totally negate the need for a native windows native toolchain. That said, this script does not (yet) build the dragon lib (libdragon), and older projects never had the need for Unix only utilities (like `dd`). I would suggest that `makefile` should be available, but caveats are added... OR the makefile build is just added in the CI script.

Reply:
The point is that n64.mk requires at least bash if, rm, mkdir and might grow even more. I don't think we can make it work without Linux tools. Installing msys is a fast way for the user to have those tools available, as far as I know. If you know another, I'm all ears.